admin/users/views.py

```diff
@@ -61,15 +61,11 @@ def delete(self, request, *args, **kwargs):
                     action_flag=flag
                 )
         except AttributeError:
```

Comment:
Moved the `update_admin_log` out of the try-block. All other operations within the if statement and the `user.save()` are either assignment statements or attribute manipulation.